packages/datepickers/src/Datepicker/Datepicker.tsx

```diff
@@ -328,7 +328,8 @@ Datepicker.defaultProps = {
   refKey: 'ref',
   animate: true,
   eventsEnabled: true,
-  zIndex: 1000
+  zIndex: 1000,
+  locale: 'en-US'
```

Comment:
Should we have a default? Like the issue mentioned in the PR if a user forgets to pass the locale in a custom formatter to match this default it'll have some confusing UX if they don't use the US format.

Reply:
This originally came up in the `#garden` channel. In some specific locale and browser combinations, `Intl.DateTimeFormat` will use the system locale by default.

This works for formatting, but browsers are only able to natively parse in short, medium, and long `en-US` locales. (There are some slight variations between browsers, but this is mostly consistent).

Without the default we would have the Datepicker showing in `danish` for example, but unable to parse anything until MomentJS or date-fns is provided. By locking it to `en-US` it ensures that the date is always able to be parsed.